deepdives/build-process/README.md

```diff
@@ -0,0 +1,66 @@
+# Deep Dive: The Build Process
+
+## Why?
+As you start to create more and more ambitious apps, the time you invest into a build process which automates work for you, cleans up your code, and gives you access to more features becomes more and more worth it. These tools can be intimidating to learn at first, but with a bit of pre work they can make your life a ton easier.
+
+## The Parts of the Process
+
+### Compilers
+You've already been exposed to these with SASS and LESS compilers. You can also use compilers (or sometimes call transpilers) to convert modern es6 or es7 javascript code into widely supported es5 javascript. The most recent updates to javascript has helped to smooth over some ugly parts (I'm looking at your Javascript OOP) and added a ton of new features. We'll get into how to use these new features in the exercises. Luckily, as long as we are just considering vanilla javascript, as opposed to coffescript or typescript, the de facto standard for compiling your code is:
+
+##### Babel
+![Babel](https://avatars0.githubusercontent.com/u/9637642?v=3&s=400)
+
+Babel can convert basically any javascript (including jsx!) you can imagine into es5. It does this by requireing users to specify a ```preset``` or set of presets that babel will use to do the compilation. You can also specify ```plugins``` to extend the features of babel. The way that you specify these is different based on the rest of the tools we use. You can look at the examples above to see how it is done in different setups.
```

Comment:
_Nitpicks_
requireing -> requiring
or set of presets -> (or a set of presets)
babel -> Babel, throughout -- for consistency

_Rewording to be more explicit_
The way that you specify... -> "The way you specify presets and plugins depends on how you choose to set up Babel. You can check out the `grunt`, `gulp`, and `webpack` examples in this directory to see how preset & plugin specification works in each of those cases.